exercises/triangle/example.rb

```diff
@@ -1,44 +1,37 @@
-class TriangleError < RuntimeError
+module BookKeeping
+  VERSION = 1
 end
 
 class Triangle
-  attr_reader :a, :b, :c
-  def initialize(a, b, c)
-    @a = a
-    @b = b
-    @c = c
-  end
+  attr_reader :sides
 
-  def kind
-    fail TriangleError if illegal?
```

Comment:
The canonical data for the triangle problem was recently [updated](https://github.com/exercism/x-common/commit/b806246cf36ee885d0aa8ad86985a752183bc3a3)  to test for properties instead of its type.  Thoughts?